pages/doc/trace_data_details.md

```diff
@@ -88,13 +88,44 @@ getAllUsers source=localhost traceId=7b3bf470-9456-11e8-9eb6-529269fb1459 spanId
 
 ### Span Tags
 
-Span tags are special tags associated with a span.
+Span tags are special tags associated with a span and are key value pairs.
 
 - **Required**. Many of the span tags are required for a span to be valid.
-- **Optional (Custom)**. An application can be instrumented to include custom span tags as well. Custom tag names must not use the reserved span tag names listed in the following tables.
-
-**Note:** The maximum allowed length for a combination of a span tag key and value is 254 characters (255 including the "=" separating key and value). If the value is longer, the span is rejected.
-
+- **Optional (Custom)**. An application can be instrumented to include custom span tags as well. Custom tag names must not use the reserved span tag names.
+{{site.data.alerts.note}}
+The following table lists the maximum characters you can assign a span tag.
+<table>
+<colgroup>
+<col width="20%" />
+<col width="10%" />
+<col width="70%" />
+</colgroup>
+<thead>
+<tr>
+<th>Element</th>
+<th>Maximum length</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>Span tag key</td>
+<td>128</td>
+<td>If the span-tag-key exceeds the maximum length, the span associated with it is blocked by Wavefront, and you won't see the span details in Wavefront.</td>
```

Comment:
why the hyphens?

...and you won't see the span details in Wavefront.
RK>> Is that useful? It's not just that I won't see the details, but they're not available. I think it's fine to stop after "blocked by Wavefront".